diff -up mono-5.18.0.268/mcs/build/profiles/basic.make.omv~ mono-5.18.0.268/mcs/build/profiles/basic.make
--- mono-5.18.0.268/mcs/build/profiles/basic.make.omv~	2019-03-02 13:45:14.264446864 +0000
+++ mono-5.18.0.268/mcs/build/profiles/basic.make	2019-03-02 13:45:47.185865742 +0000
@@ -52,7 +52,7 @@ LIBRARY_COMPILE = $(BOOT_COMPILE)
 #
 # Copy from rules.make because I don't know how to unset MCS_FLAGS
 #
-USE_MCS_FLAGS = /codepage:$(CODEPAGE) /nologo /noconfig /deterministic $(LOCAL_MCS_FLAGS) $(PLATFORM_MCS_FLAGS) $(PROFILE_MCS_FLAGS) $(MCS_FLAGS)
+USE_MCS_FLAGS = /codepage:$(CODEPAGE) /parallel- /langversion:latest /nologo /noconfig /deterministic $(LOCAL_MCS_FLAGS) $(PLATFORM_MCS_FLAGS) $(PROFILE_MCS_FLAGS) $(MCS_FLAGS)
 
 .PHONY: profile-check do-profile-check
 profile-check:
diff -up mono-5.18.0.268/mcs/build/profiles/build.make.omv~ mono-5.18.0.268/mcs/build/profiles/build.make
diff -up mono-5.18.0.268/mcs/build/rules.make.omv~ mono-5.18.0.268/mcs/build/rules.make
--- mono-5.18.0.268/mcs/build/rules.make.omv~	2019-03-02 13:45:54.098163656 +0000
+++ mono-5.18.0.268/mcs/build/rules.make	2019-03-02 13:46:03.994590183 +0000
@@ -34,7 +34,7 @@ BUILD_TOOLS_PROFILE = build
 endif
 endif
 
-USE_MCS_FLAGS = /codepage:$(CODEPAGE) /nologo /noconfig /deterministic $(LOCAL_MCS_FLAGS) $(PLATFORM_MCS_FLAGS) $(PROFILE_MCS_FLAGS) $(MCS_FLAGS)
+USE_MCS_FLAGS = /codepage:$(CODEPAGE) /parallel- /langversion:latest /nologo /noconfig /deterministic $(LOCAL_MCS_FLAGS) $(PLATFORM_MCS_FLAGS) $(PROFILE_MCS_FLAGS) $(MCS_FLAGS)
 USE_MBAS_FLAGS = /codepage:$(CODEPAGE) $(LOCAL_MBAS_FLAGS) $(PLATFORM_MBAS_FLAGS) $(PROFILE_MBAS_FLAGS) $(MBAS_FLAGS)
 USE_CFLAGS = $(LOCAL_CFLAGS) $(CFLAGS) $(CPPFLAGS)
 CSCOMPILE = $(Q_MCS) $(MCS) $(USE_MCS_FLAGS)
